pybombs-commondeps/Dockerfile

```diff
@@ -1,16 +1,26 @@
-FROM pybombs/pybombs-prefix:2.3.3
+FROM pybombs/pybombs-prefix:2.3.4
 LABEL maintainer=martin@gnuradio.org
 
 # This allows setting the makewidth temporarily to a higher value
 ARG makewidth=2
 
+# Temporarily set frontend to noninteractive to avoid warnings
+ARG DEBIAN_FRONTEND=noninteractive
```

Comment:
Is this needed? It looks like you're not using the argument.

Reply:
I tested with these lines removed and everything still worked fine.

Reply:
Its a "nice to have" as in it won't break with it removed, but if you look at the apt-get output it throws a ton of warnings here that make it look like something is going wrong... I prefer to have it in there (ARG makes it only active at build-time) but I'm not insisting on it if it doesn't break w/o.

Reply:
Why does setting a Docker `ARG` have any effect though?

Reply:
You're right, it looks redundant given line 19... Not very clean to have it in there twice, I'll take it out.